.github/workflows/build-conda-windows.yml

```diff
@@ -34,21 +34,19 @@ jobs:
             smoke-test-script: test/smoke_test.py
             package-name: torchvision
     name: ${{ matrix.repository }}
-    uses: pytorch/test-infra/.github/workflows/build_wheels_windows.yml@main
+    uses: pytorch/test-infra/.github/workflows/build_conda_windows.yml@main
     with:
+      conda-package-directory: ${{ matrix.conda-package-directory }}
       repository: ${{ matrix.repository }}
       ref: ""
       test-infra-repository: pytorch/test-infra
       test-infra-ref: main
       build-matrix: ${{ needs.generate-matrix.outputs.matrix }}
       pre-script: ${{ matrix.pre-script }}
-      env-script: ${{ matrix.env-script }}
       post-script: ${{ matrix.post-script }}
       package-name: ${{ matrix.package-name }}
-      smoke-test-script: ${{ matrix.smoke-test-script }}
```

Comment:
Why are we removing the smoke test here? Both Linux and macOS conda builds have it.

Reply:
@pmeier looks like smoke tests for win conda is not integrated yet. Here is the PR to do this: https://github.com/pytorch/test-infra/pull/4095

Reply:
Let's merge that first so we can be sure here that our conda build is running properly.